controllers/seedgen_controller.go

```diff
@@ -413,7 +413,7 @@ func (r *SeedGeneratorReconciler) getRecertImagePullSpec(seedgen *seedgenv1alpha
 func (r *SeedGeneratorReconciler) pullRecertImagePullSpec(seedgen *seedgenv1alpha1.SeedGenerator) error {
 	recertImage := r.getRecertImagePullSpec(seedgen)
 
-	_, err := r.Executor.Execute("podman", "pull", recertImage)
+	_, err := r.Executor.Execute("podman", "pull", "--authfile", seedgenAuthFile, recertImage)
```

Comment:
I think you would need to get the .dockerconfigjson from the openshift-config/pull-secret secret and merge it with the seedgenAuth before writing the file. 

I guess I should have defined this with "--authfile /var/lib/kubelet/config.json", which should be the pull secrets for the cluster.

Reply:
Ok, for the moment, I think we can keep the fix simple by just adding the `"--authfile /var/lib/kubelet/config.json"` suggested... in any case, seems like a minor issue.

if a new / different creds is required for the recert image in the future, we can write the code for it